src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -439,6 +439,18 @@ public void testSetters() {
         }
     }
 
+    @SuppressForbidden
```

Comment:
please add comment about a reason why we have `SuppressForbidden` , verify is one of many methods we forbid, so reader should know what was out target.

Reply:
Sorry but can you tell me the reason why we should avoid using verify() method? I don't know what to write in the comment

Reply:
Reason is described at https://github.com/checkstyle/checkstyle/issues/11446

Make a comment like:
```
/*
SuppressForbidden is used to allow usage of 'verify' method. We should use it until we add support for Checker properties in inlined config 
*/
```

Reply:
okay and I forgot to mention that there are other test methods too in CheckerTest which uses `verify()` but does not have  `@SuppressForbidden` annotation and also don't have the comment you asked me to add... should I add the annotation and comment to those methods too?

Reply:
To unblock this PR sooner please do only update for test you updating.

We can create separate PR to update all others to  be in better form.

Reply:
We can make to use good verify method if we define config by xml , example https://github.com/checkstyle/checkstyle/commit/3e1be5063f621252da86a09316fe31a7209ea366#diff-00c2dc5d7847e5fac63bbc22c7910ed1d043fc47999674cb2f11c53db985d514R2

In this case we do not need any suppression at all, and life becomes logical and simple. Please try this.

Reply:
@romani I tried using `verifyWithInlineConfigParser()` in place of `verify()` as showed in the commit, but the test started failing idk why. Here is the test case:

```java
@Test
public void testSetSeverity() throws Exception {
    final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;

    verifyWithInlineConfigParser(getPath("InputCheckerTestSeverity.java"), expected);
} 
```

**Error log:**

```
com.puppycrawl.tools.checkstyle.api.CheckstyleException: Config comment not specified properly in C:\Open Source\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checker\InputCheckerTestSeverity.java

	at com.puppycrawl.tools.checkstyle.bdd.InlineConfigParser.parse(InlineConfigParser.java:210)
	at com.puppycrawl.tools.checkstyle.bdd.InlineConfigParser.parse(InlineConfigParser.java:190)
	at com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport.verifyWithInlineConfigParser(AbstractModuleTestSupport.java:235)
	at com.puppycrawl.tools.checkstyle.CheckerTest.testSetSeverity(CheckerTest.java:446)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:728)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:218)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:214)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:139)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:69)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)
Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: Unable to resolve module name: LineLength. Please check for spelling errors or specify fully qualified class name.
	at com.puppycrawl.tools.checkstyle.bdd.InlineConfigParser.getFullyQualifiedClassName(InlineConfigParser.java:331)
	at com.puppycrawl.tools.checkstyle.bdd.InlineConfigParser.setModuleName(InlineConfigParser.java:391)
	at com.puppycrawl.tools.checkstyle.bdd.InlineConfigParser.handleXmlConfig(InlineConfigParser.java:277)
	at com.puppycrawl.tools.checkstyle.bdd.InlineConfigParser.setModules(InlineConfigParser.java:250)
	at com.puppycrawl.tools.checkstyle.bdd.InlineConfigParser.parse(InlineConfigParser.java:207)
	... 71 more


Process finished with exit code -1
```

Should we use `verify()` method instead of `verifyWithInlineConfigParser()`? The test case was working fine when we were using `verify()` method.

Reply:
Please push your code to let me see all details

Reply:
sorry for the late reply, I have pushed the changes @romani

Reply:
@Zopsss severity is a property of the Checker module not line Length I think that's why it says config comment isn't specified properly

Reply:
> @Zopsss severity is a property of the Checker module not line Length I think that's why it says config comment isn't specified properly

thats not true. Every module has this property, the implementation of this property is written at Checker check. Every modules are child of Checker so they can use this property to control audit events. Check out the [doc](https://checkstyle.sourceforge.io/config.html#Severity) for more info.